### PR TITLE
doc: remove legacy template file

### PR DESCRIPTION
**Pull request by tovogt**
Changes proposed in this PR:
- This PR polishes one last detail that remains to do from the great work by @peanutfun in #600: Namely, there was a misplaced header left from a legacy template file.

image

### PR Author Checklist

- [x] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [x] Source branch up-to-date with target branch
- [x] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- [x] [Tests][testing] updated
- [x] [Tests][testing] passing
- [x] No new [linter issues][linter]

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/stable/guide/Guide_Continuous_Integration_and_Testing.html#3.C.--Static-Code-Analysis

### CODE REVIEW

File: climada/util/coordinates.py [hunk 1]
```diff
@@ -290,22 +290,26 @@ def dist_approx(lat1, lon1, lat2, lon2, log=False, normalize=True,
...
+        are reparametrized accordingly using :py:func:`lon_normalize`. Note that this option has
+        no effect when using the "geosphere" method because it is independent from the
+        parametrization. Default: True
     method : str, optional
         Specify an approximation method to use:
-            * "equirect": Distance according to sinusoidal projection. Fast, but inaccurate for
-              large distances and high latitudes.
-            * "geosphere": Exact spherical distance. Much more accurate at all distances, but slow.
+
+        * "equirect": Distance according to sinusoidal projection. Fast, but inaccurate for
```
Comment (tovogt) [inline climada/util/coordinates.py:300]: This is unrelated to the main issue in this PR. Correcting indentation and adding empty lines before and after bullet lists fixes a few sphinx warnings.

File: doc/climada/climada.util.rst [hunk 2]
```diff
@@ -9,14 +9,6 @@ climada\.util\.api\_client module
...
     :show-inheritance:
 
-climada\.util\.cached module
-----------------------------
-
-.. automodule:: climada.util.cached
-    :members:
-    :undoc-members:
-    :show-inheritance:
-
```
Comment (tovogt) [inline doc/climada/climada.util.rst:19]: This submodule doesn't exist, hence we can remove the reference from the docs.

File: doc/guide/Guide_Git_Development.ipynb [hunk 3]
```diff
@@ -179,7 +179,7 @@
    "source": [
     "### Gitflow works on the `develop` branch instead of `main`\n",
     "\n",
-    "![](flow_1.png)\n",
+    "![](img/flow_1.png)\n",
```
Comment (tovogt) [inline doc/guide/Guide_Git_Development.ipynb:182]: This fixes an error/warning output of sphinx.

**Review by tovogt — COMMENTED** [review]
Fixed most (all?) of the sphinx warnings while I was at it...

File: climada/util/coordinates.py [hunk 4]
```diff
@@ -290,22 +290,26 @@ def dist_approx(lat1, lon1, lat2, lon2, log=False, normalize=True,
         If False, assume that all longitudinal values lie within a single interval of size 360
         (e.g., between -180 and 180, or between 0 and 360) and such that the shortest path between
         any two points does not cross the antimeridian according to that parametrization. If True,
-        a suitable interval is determined using `lon_bounds` and the longitudinal values are
-        reparametrized accordingly using `lon_normalize`. Note that this option has no effect when
-        using the "geosphere" method because it is independent from the parametrization.
-        Default: True
+        a suitable interval is determined using :py:func:`lon_bounds` and the longitudinal values
```
[the remark this answers is not in the document]
Comment (emanuel-schmid) [reply climada/util/coordinates.py:293]: ... which doesn't make anything, right?

**Comment by emanuel-schmid** [comment]
looks good to me. 😃

File: climada/util/coordinates.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (tovogt) [reply climada/util/coordinates.py:293]: Before:
image
After:
image

[the remark this answers is not in the document]
Comment (emanuel-schmid) [reply climada/util/coordinates.py:293]: Of course it does! Sorry. (I looked at the wrong tab 🙄)

**Comment by peanutfun** [comment]
Excellent, @tovogt! I was wondering about this stray title 😅 

Also, nice work with the crossrefs. Most of the docstrings don't use them yet. That might be a job for a student assistant 🤔
